components/delete_channel_modal/delete_channel_modal.test.tsx

```diff
@@ -56,7 +64,7 @@ describe('components/delete_channel_modal', () => {
         );
 
         wrapper.setState({show: true});
-        wrapper.instance().onHide();
+        wrapper.instance().onHide(); // Property 'onHide' does not exist on type 'Component<{}, {}, any>'
```

Comment:
As noted in the comments, I'm getting a TypeScript error in VSCode: `Property 'onHide' does not exist on type 'Component<{}, {}, any>'`. I noticed a couple other components made `onHide` a public method in the actual component with a note that it was required for tests, but even after doing that I was still getting the same error.

Reply:
You need to specific a type for the generic `shallow()` method, since it can't autodetect.

Something like:
```
const wrapper = shallow<DeleteChannelModal>(
    <DeleteChannelModal {...baseProps}/>
);
```

Reply:
Ah, got it. Thanks!